client/src/containers/CovidPage/CovidPage.jsx

```diff
@@ -144,6 +157,18 @@ class CovidPage extends Component {
 
   filterTime = (time) => {};
 
+  handleJoyrideCallback = data => {
+    const { action, index, status, type } = data;
+
+    if ([EVENTS.STEP_AFTER].includes(type)) {
+      this.setState({ run: false });
+      const context = this.context;
+      context.enabled = false;
+      context.stepIndex = 0;
+      context.run = false
+    }
+  };
```

Comment:
if the user ends the tutorial, exit the joyride and make sure joyride won't start again when covid page is opened